### PR TITLE
Update members.js

### PR DESCRIPTION
**Pull request by beejay777**

### CODE REVIEW

**Comment by axispx** [comment]
Can you add the `// add here` comment at the end so that others can know where to add their data?

File: members/members.js [hunk 1]
```diff
@@ -149,7 +149,17 @@ const members = [
...
+  {
+    fullName: 'Bijay Pathak',
+    discordUsername: 'aquilabijay#4800',
+    about: 'Engineering Student, looking forward to contribute',
+    links: {
+        github: 'beejay777', // required
+        twitter: '',
+        facebook: ''
+    }
+},
```
Comment (beejay777) [inline members/members.js]: // add here

**Comment by axispx** [comment]
also, instead of giving empty links you can skip the data field

**Comment by axispx** [comment]
can you change `//Add here` to `// add here` to be consistent with the style format?

**Comment by beejay777** [comment]
okay :)

**Comment by beejay777** [comment]
Hope its all okay now :)

**Comment by axispx** [comment]
looks good to me

**Comment by axispx** [comment]
you have missed a comma before adding your data object which is causing an error and you have to provide indentation of 2 spaces

**Pull request merged**